Exercise4.js

```diff
@@ -9,8 +9,7 @@ function findEvenNumbers(numbers) {
   return evenNumbers;
 }
 
-*/ 
+*/
 
-function findEvenNumbers(numbers){
-    return numbers.filter(number=> number%2=== 0)
-}
+const findEvenNumbers = (numbers) =>
```

Comment:
When arrow functions have a single parameter the brackets can be omitted to increase code readability.

Read more about arrow functions : [Omitting brackets](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)